code/src/components/TaskPage.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react"
```

Comment:
You have nicely separated all the components, so the code stays clear. Next project you could also consider to make folder called "pages" and put there your different pages (in this case start/main/error) and then keep the different components in the component file, so it would be even more clear.

Reply:
Nice tip! 😊